grossmont/computer-science-and-cybersecurity-organization/sections/header/header.html

```diff
@@ -24,32 +24,29 @@
         <a class="tab" href="/grossmont/computer-science-and-cybersecurity-organization/pages/contact/contact.html"
           target="_parent">Contact</a>
         <!-- <a class="tab" href="">Activites</a> -->
+        <img id="icon" src="/grossmont/computer-science-and-cybersecurity-organization/media/moon.png">
       </nav>
     </div>
     <!-- <header> -->
     <div class="imageText">
       <div class="image">
 
-        <a class="grossmont-logo" href="/index.html" target="_parent">
-          <img src="/grossmont/computer-science-and-cybersecurity-organization/media/Logo (no words).png" width="100"
-            height="100">
+        <a href="/index.html" target="_parent">
+          <img id="clubLogo"
+            src="/grossmont/computer-science-and-cybersecurity-organization/media/grossmont CS & Cyber club logo only.png"
+            width="100" height="100">
```

Comment:
Changed the image to the one without background, so the header background color is visible through it. I also created the clubLogo id, which replaced the grossmont-logo class which wasn't used, and used it to scale down the club logo in header.css